pkg/provisioner/ironic/ironic.go

```diff
@@ -1784,6 +1790,14 @@ func (p *ironicProvisioner) softPowerOff() (result provisioner.Result, err error
 	return result, nil
 }
 
+func (p *ironicProvisioner) hostName() string {
```

Comment:
I'd suggest to rename this method to something like ie `getIronicNodeNameFromHost(host metal3v1alpha1.BareMetalHost)` to make it more clear that this is mapping function to calculate the effective Ironic node name to be used given a BMH (since the current name sounds a little bit too vague for me)

Reply:
I'll meet you half way, I still need p.watchAllNamespaces, so I'll this
```
func (p *ironicProvisioner) IronicNodeNameFromHost() string
```
also golang getters shouldn't start with "get" > https://golang.org/doc/effective_go#Getters

Reply:
Mine was just an example, `IronicNodeNameFromHost` it's fine for me :+1: